migrations/html/v3/posthtml-forge-button.mjs

```diff
@@ -112,6 +112,33 @@ export default function transform(tree) {
 
     return node;
   });
+
+  // Move nested tooltips
+  tree.match([
+    { tag: 'forge-button' },
+    { tag: 'forge-icon-button' },
+    { tag: 'forge-fab' }
+  ], node => {
+    const nestedTooltip = findChildNode(node, child => child.tag === 'forge-tooltip');
+    if (!nestedTooltip) {
+      return node;
+    }
+
+    // Copy all attributes from the nested button to the forge button
+    if (nestedButton.attrs) {
+      node.attrs = {
+        ...node.attrs,
+        ...nestedButton.attrs,
+      };
+    }
```

Comment:
We probably don't need this since it would be handled by the other migration?

Reply:
Ah yeah, I copied from the other migration and forgot to remove this